skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +458,26 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        if origin is None:
+            origin = shape[0]//2, shape[1]//2
```

Comment:
I think it can't hurt, so it would be fine with me.  People in principle could run this algorithm on a region-of-interest which had an odd number of rows/cols, for example.  I'm happy to do a PR or you can.  Totally up to you...